docker-dev.env

```diff
@@ -10,3 +10,7 @@ SECRET_KEY_BASE=4f13a3e80905baa2e8e35489ebfae43d265cf8ca92bcac80e1ed0d2265874765
 REDIS_URL=redis://redis:6379
 PUMA_OPTIONS=
 PUBLIC_REGISTRATION=true
+RECAPTCHA_SITE_KEY=6LdCLQMjAAAAAOyRvpHlZ7hgHE8JHDdshNYms8zo
+RECAPTCHA_SECRET_KEY=6LdCLQMjAAAAAOy0aoCWAzTs0mo-0PYw9nd4kwyS
+RECAPTCHA_SITE_KEY_V3=6LeLkv0iAAAAAO51qtrpPv1UVVhZYYIsaBqyl6JQ
+RECAPTCHA_SECRET_KEY_V3=6LeLkv0iAAAAAAZi7HvBkxPW-BDcigTDJQtdj0iV
```

Comment:
Security Rule: NEVER commit credentials, it's unsafe even for private repositories.

Don't forget to invalidate and regenerate these keys.

```suggestion
```